### ISSUE TITLE
[FEATURE] Can I adjust the item drop filter position?

### ISSUE THREAD

**Issue by arithxp** [soarqin/D2RMH#41]
In the current version, if I use it as Right + Map Center, it is displayed too large in Arcane Sanctuary.
So I modified the size on the source, but doing so cuts the drop item filter.
Is it possible to place the drop filter below the map? 

maprenderer.cpp
```
    case 3:
        renderer_.owner()->move(d2rRect.left + 1280, d2rRect.top + 38, 640, 400 - 38);
        w = (float)640;
        h = (float)400-38;
        break;
```
D2RMH.ini
```
position = 3
scale=2.22
msg_position = 0.95,0.7,2
```

And do you have any plans to hide the map when switch to the inventory, skill tree, or another window? 

edit: hide map when switch other windows
maprenderer.cpp (line 76)
```
    d2rProcess_.updateData();
    if (!d2rProcess_.available()) {
        enabled_ = false;
        return;
    }
    if (d2rProcess_.hwnd() != GetForegroundWindow()) {
        enabled_ = false;
        return;
    }
    switch (cfg->show) {
```

**Comment by soarqin** [comment]
hmmm, currently you can set a smaller `msg_font_size` to make more rows shown
I will add config entries to restrict map size in next release and make msg_position relative to whole D2R window

**Comment by arithxp** [comment]
> 
> 
> hmmm, currently you can set a smaller `msg_font_size` to make more rows shown I will add config entries to restrict map size in next release and make msg_position relative to whole D2R window

I will use a reduced font size until a revised version is released. thank you.

**Comment by soarqin** [comment]
added in last commit, you can test if it works fine

**Comment by Jarboy87** [comment]
Been playing around with this also, the text can only be drawn within the overlay itself which is generally taken up by the map. Is it possible to create a second overlay area dedicated to an item list? that can be enabled and moved independent from the map? By the way, excellent work guys, I was surprised how quickly you were able to get the item filter available.

**Comment by soarqin** [comment]
> Been playing around with this also, the text can only be drawn within the overlay itself which is generally taken up by the map. Is it possible to create a second overlay area dedicated to an item list? that can be enabled and moved independent from the map? By the way, excellent work guys, I was surprised how quickly you were able to get the item filter available.

already implemented, you can compile the latest codes in master branch to see if it works fine

**Comment by arithxp** [comment]
> 
> 
> added in last commit, you can test if it works fine

![image](https://user-images.githubusercontent.com/6009737/142133658-3412c833-db5c-4d67-bb38-72433d5d4e90.png)

It was hard to fine tune, but it works good.

**Comment by soarqin** [comment]
> And do you have any plans to hide the map when switch to the inventory, skill tree, or another window?
implemented in 0e42d9e
